### PR TITLE
Remove output redirection to resolve stuck behavior in pod-install

### PR DESCRIPTION
**Pull request by M-ZubairAhmed**
#### Summary
The purpose of the `&> /dev/null` redirection was to suppress output, including error messages and standard output. However, this redirection caused an issue where the command appeared to get stuck until pressing ["Enter" to proceed](https://community.mattermost.com/core/pl/8znnbqkwujgrxx61zxuis7t9tc). This behaviour might be due to buffering, where the output is not immediately displayed but is flushed upon pressing "Enter." It could be that there may not be any problem with the script itself; rather, it gave the perception of being stuck. Also, it seems that Cocoapods installation takes [a long time](https://stackoverflow.com/questions/23755974/cocoapods-pod-install-takes-forever). Given these factors, it is more reasonable to remove the output redirection and run the script with the complete output.

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: 

#### Screenshots

image

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: scripts/postinstall.sh [hunk 1]
```diff
@@ -1,20 +1,20 @@
 #!/usr/bin/env bash
 
-function ios() {
+function getCocoaPodsFormacOS() {
```
Comment (enahum) [inline scripts/postinstall.sh:3]: `installPods` reads much better

File: scripts/postinstall.sh [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
     echo "Getting Cocoapods dependencies"
-    npm run pod-install &> /dev/null
+    npm run pod-install
 }
 
-function iosM1() {
+function getCocoaPodsFormacOSM1() {
```
Comment (enahum) [inline scripts/postinstall.sh:8]: `installPodsM1`

**Review by enahum — CHANGES_REQUESTED** [review]
I'm not against this although the output is long and horrible

If this is the case, shouldn't this also be done for the Gem installation with bunlder?

**Comment by M-ZubairAhmed** [comment]
> If this is the case, shouldn't this also be done for the Gem installation with bunlder?

Didnt notice it was also used in `build.sh`, will change

File: scripts/postinstall.sh (hunk 1 above)
Comment (M-ZubairAhmed) [reply scripts/postinstall.sh:3]: Done ✅

File: scripts/postinstall.sh (hunk 2 above)
Comment (M-ZubairAhmed) [reply scripts/postinstall.sh:8]: Done ✅

**Review by enahum — APPROVED** [review]

**Pull request merged**